bin/classmap_generator.php

```diff
@@ -228,7 +231,11 @@
     $maxWidth = max($maxWidth, strlen($match[1]));
 }
 
-$content = preg_replace('(\n\s+([^=]+)=>)e', "'\n    \\1' . str_repeat(' ', " . $maxWidth . " - strlen('\\1')) . '=>'", $content);
+$content = preg_replace_callback('(\n\s+([^=]+)=>)', function ($matches)
+{
```

Comment:
psr-2 coding standard, the brace <code>{</code> should be in same line after <code>function ($matches)</code>, so it should be : 
```php
$content = preg_replace_callback('(\n\s+([^=]+)=>)', function ($matches) {
```